packages/libs/pactjs-generator/package.json

```diff
@@ -21,23 +21,11 @@
     "build:grammar:watch": "chokidar src/lexer.js src/grammar.ne src/tests/test.contract.pact -c \"rushx build:grammar\"",
     "test:grammar:watch": "heft test -w -t grammar --disable-code-coverage",
     "util:lexer-grammar:watch": "concurrently --kill-others npm:build:*:watch # npm:test:grammar:watch",
+    "format": "prettier config src --write -u --config ../../../.prettierrc --ignore-path ../../../.prettierignore",
```

Comment:
Can we provide the `.prettierrc` via a `@kadena-dev` package? Similar to eslint-config? Or are there other reasons to have it exposed as this?

Reply:
Good point. For now I just repeated what we already did. Not sure what's the best way here. Ideally, packages are fully self-contained so I guess we could copy that file around. Or put it in another package, but I don't like the magic here (e.g. symlinks or copying files before running scripts, etc). Could be something to discuss and apply in another PR. Current solution is ugly but 100% concise + effective.